client/src/locales/fr-FR.json

```
@@ -1542,12 +1542,12 @@
             "information": "Informations",
             "startingDate": "Date de début",
             "endingDate": "Date de fin",
-            "members": {
-                "title": "Licence membre | Licences membres",
-                "admin": "Membre administrateur | Membres administrateurs",
+            "users": {
+                "title": "Licence utilisateur | Licences utilisateurs",
```

I don't understand plural for "utilisateur" here
```suggestion
                "title": "Licence utilisateur | Licences utilisateur",
```